elvis.config

```diff
@@ -27,7 +27,7 @@
                  %% Allow usage of ?assertMatch and similar macros, since after expanding they
                  %% result in use of variables like __X and __V that wouldn't be permitted by
                  %% default regex.
-                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z_]*)$"}}
+                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z]*)$"}}
```

Comment:
I think we do need `_` in this regexp. Otherwise Elvis may complain about macros like `?A_MACRO_WITH_WORDS`, am I right? We have such macros in tests.